repos/system_upgrade/el7toel8/libraries/overlaygen.py

```diff
@@ -7,6 +7,61 @@
 from leapp.libraries.stdlib import CalledProcessError, api, run
 
 
+def _ensure_enough_diskimage_space(space_needed, directory):
+    stat = os.statvfs(directory)
+    if (stat.f_frsize * stat.f_bavail) < (space_needed * 1024 * 1024):
+        message = 'Not enough space available for creating required disk images. Needed: {} MiB'.format(space_needed)
+        api.current_actor().error(message)
```

Comment:
```python
Process Process-175:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib64/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 63, in _do_run
    target_actor(logger=logger, messaging=messaging, config_model=config_model).run(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/leapp/actors/__init__.py", line 320, in run
    self.process(*args)
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/actors/targetuserspacecreator/actor.py", line 29, in process
    userspacegen.perform()
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py", line 168, in perform
    xfs_info=xfs_info) as overlay:
  File "/usr/lib64/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/libraries/overlaygen.py", line 185, in create_source_overlay
    mounts = _prepare_required_mounts(scratch_dir, mounts_dir, _get_mountpoints(storage_info), xfs_info)
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/libraries/overlaygen.py", line 38, in _prepare_required_mounts
    _ensure_enough_diskimage_space(space_needed, scratch_dir)
  File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/libraries/overlaygen.py", line 16, in _ensure_enough_diskimage_space
    api.current_actor().error(message)
AttributeError: 'TargetUserspaceCreator' object has no attribute 'error'
```

I guess you wanted to use "report_error" here...

Reply:
actually no, current_logger was what I wanted